techeria_controller/techeria_app/models.py

```diff
@@ -38,60 +36,62 @@ class Meta:
 class SellerModel(models.Model):
     first_name = models.CharField(max_length=255)
     last_name = models.CharField(max_length=255)
-    username = models.CharField(max_length=255)
+    user_name = models.CharField(max_length=255)
     date_of_birth = models.DateField()
     email = models.EmailField()
-    mobile_number = models.IntegerField()
+    mobile_number = models.IntegerField(max_length=10)
     address = models.CharField(max_length=255)
     city = models.CharField(max_length=255)
     state = models.CharField(max_length=255)
     zip_code = models.CharField(max_length=255)
     country = models.CharField(max_length=20)
-    
+
 
     class Meta:
         db_table = "seller"
 
 
 class Laptops(models.Model):
     # id = models.BigIntegerField()
-    image = models.CharField(max_length=100)
+    image = models.ImageField(null=False, blank=False, upload_to='images/')
     name = models.CharField(max_length=255)
     price = models.FloatField()
     description = models.TextField()
     category = models.CharField(max_length=255)
 
     class Meta:
-        managed = False
+
         db_table = 'laptops'
 
-   
+class Smartphone(models.Model):
```

Comment:
This needs to be unit tested to make sure that this functionality is working correctly.

Reply:
yeah I had included unit testing